.ci/run.sh

```diff
@@ -64,5 +64,6 @@ check_log_files()
 export RUNTIME="kata-runtime"
 
 sudo -E PATH="$PATH" bash -c "make check"
+sudo -E PATH="$PATH" bash -c "make test"
```

Comment:
Could you invert the order here:
```bash
sudo -E PATH="$PATH" bash -c "make test"
sudo -E PATH="$PATH" bash -c "make check"
```
No need to run the full CI if `checkcommits` does not pass.

Reply:
Well, `checkcommits` isn't actually being *run* - we're just running its unit tests.

But I am wondering if the tests `Makefile` need a tweak so that:

- `test`: runs the integration tests.
- `check`: runs the checkers (and self-checks on commands that can be built in the repo):
  - `checkcommits` unit-tests.
  - `log-parser` unit-tests.

wdyt @chavafg?

Reply:
@jodh-intel I think the tweak make sense. Want to add it in this PR?

Reply:
Thanks @chavafg. Branch updated.